test/util/common.go

```diff
@@ -355,7 +355,7 @@ func MakeTestMarshaler() codec.Codec {
 }
 
 // SetupFiveValChain does all the initialization for a 5 Validator chain using the keys here
-func SetupFiveValChain(t *testing.T) (TestInput, sdk.Context) {
+func SetupFiveValChain(t *testing.T, register bool) (TestInput, sdk.Context) {
```

Comment:
Why would we have an option to register? All validators should register an EVM address, no?